projects/core/components/scroll-controls/scroll-controls.template.html

```diff
@@ -4,6 +4,7 @@
         class="bar bar_vertical"
         [class.bar_has-horizontal]="bars[1]"
         [@tuiFadeIn]
+        tuiScrollbarWrapper
```

Comment:
Please move the directive to the line 4. We try to maintain certain order of attributes across our templates. I need to check if Prettier supports it these days :)

Reply:
Done.
Did you manage to check for Prettier support? I'm wondering if this is possible )